apps/web/public/mockServiceWorker.js

```diff
@@ -0,0 +1,307 @@
+/* eslint-disable */
```

Comment:
Where is this file coming from? Who created it and why it is in the public folder? Isn't this related to testing?